packages/pipeline/src/scripts/pull_erc20_events.ts

```diff
@@ -1,10 +1,10 @@
-// tslint:disable:no-console
```

Comment:
As per our discussion in the last Eng All Hands, we're going to switch to using `logUtils` for scripts in the pipeline package. I went ahead and did that for this file since I was working on it anyways. Planning to do a pass through all the other scripts in a future PR.